src/Writer/XLSX/Options/WorkbookProtection.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenSpout\Writer\XLSX\Options;
+
+final class WorkbookProtection
```

Comment:
```suggestion
final readonly class WorkbookProtection
```
1. Please mark the class as `readonly`
2. Make all the properties `public` so we can get rid of the setters/getters